lib/internal/Magento/Framework/Mail/Template/TransportBuilderByStore.php

```diff
@@ -8,6 +8,13 @@
 
 use Magento\Framework\Mail\MessageInterface;
 
+/**
+ * Class TransportBuilderByStore
+ *
+ * @deprecated The ability to set From address based on store is now available
+ * in the \Magento\Framework\Mail\Template\TransportBuilder class
+ * @see \Magento\Framework\Mail\Template\TransportBuilder::setFromByStore
```

Comment:
Should be changed to `setFromByScope`

Reply:
Well spotted @buskamuza 

Changes committed.